automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -xe
+
+# This script should be able to execute KubeMacPool
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-kubemacpool-functests.sh
+
+teardown() {
+    echo "make cluster-down"
```

Comment:
echo?

Reply:
Forgot to remove it while debugging sorry :x